src/tests/test.py

```diff
@@ -28,9 +27,22 @@ async def before_after(coro, *args, **kwargs):
     return "<before>" + (await coro(*args, **kwargs)) + "<after>"
 
 
+def asyncio_run(awaitable):
+    """asyncio.run for Python 3.5"""
+    try:
+        from asyncio import run
+        aio_run = run
+    except ImportError:
+        # Python 3.5
+        from asyncio import get_event_loop
+        return get_event_loop().run_until_complete(awaitable)
+    else:
+        return aio_run(awaitable)
```

Comment:
I guess the `aio_run = run` was meant to define the runner on both try / except branches, right?
If you're not going to do it anymore, there's no need for the aio_run alias, let's drop it.